main_ovs.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/env python
+import nltk
+import re
+import string
+import json
+
+
+class FileReader:
+    @staticmethod
+    def read_file(name):
+        """ func to read from file """
+        with open(name) as fl:
+            return fl.readlines()
+
+
+class JsonWriter:
+    @staticmethod
+    def write_json(name, jsn):
+        with open(name, 'w') as file:                   # serialization
+            json.dump(jsn, file)
+
+
+class RemoverUnwantedWords:
+    @staticmethod
+    def remove_words(expr, line):
+        """ func for cleaning by regex """
+        return re.sub(expr, '', line)
+
+
+class CheckerForMeaning:
+    @staticmethod
+    def find_trash(clean_line, inf_dict):
```

Comment:
The idea to mutate the data passed as an argument is as bad as mutating the internal state. That means, instead of doing

```python
def find_trash(clean_line, inf_dict):
# ...
inf_dict['orphan_tokens'].append(i)
# ...
```

a much better strategy could be

```python
def find_trash(clean_line):
# ...
    return clean_tokens, orphan_tokens

# somewhere higher the call stack
clean_tokens, orphan_tokens = find_trash
response["orphan_tokens"] = orphan_tokens
```

*Don't* ever mutate the data passes to your methods as an argument.